convey/conveymetric/metric.go

```diff
@@ -0,0 +1,50 @@
+package conveymetric
+
+import (
+	"github.com/Comcast/webpa-common/convey"
+	"github.com/go-kit/kit/metrics"
+)
+
+// UnknownLabel is a constant for when key/tag can not be found in the C JSON
+const UnknownLabel = "unknown"
+
+// MetricClosure will be returned after update of struct, this should be used to update the struct, aka decrement the count
+type MetricClosure func()
+
+// CMetric provides
+type CMetric interface {
```

Comment:
This "stutters".  Client code refers to this type as `conveymetric.CMetric`.  Unlike other languages, you can't omit the package name when referring to the type.

It's customary in golang for a central interface in a package that would otherwise share the name of the package to be called `Interface`, e.g. `type Interface interface { ... }`.  Alternatively, you can use a single appropriate letter, e.g. `type C interface {...}`.  I personally prefer `Interface` in this particular case, but that's just a preference.

This concept is discussed at some length here:
https://golang.org/doc/effective_go.html#package-names